src/libs/state.ts

```diff
@@ -15,11 +18,16 @@ export const authStore = Utils.atom<any>({
   fenceStatus: {},
   cookiesAccepted: undefined,
   authContext: undefined,
-  oidcConfig: {},
+  oidcConfig: {
+    authorityEndpoint: undefined,
+    clientId: undefined,
+  },
   isAzurePreviewUser: undefined,
+  sessionId: undefined,
+  sessionStartTime: {} as number,
 });
 
-export const getUser = () => authStore.get().user;
+export const getUser: any = () => authStore.get().user;
```

Comment:
```suggestion
export const getUser = () => authStore.get().user;
```

Similar to an earlier comment, the inferred return type here is already `any`. Since that works for our purposes, let's stick with it instead of adding any more explicit `any`s than necessary.